e2e/tests/features/203-delay-block.spec.ts

```diff
@@ -27,7 +27,7 @@ test('delay blocks add time to events', async ({ page }) => {
   await page.getByText('New start 00:12').click();
 
   // make negative delay
-  await page.getByText('Subtract time').click();
+  await page.getByTestId('subtract-time').click();
```

Comment:
question: we should be able to get it by the label? maybe I am missing something?

Reply:
It wasn't working for some reason...

Reply:
😅 maybe check with the playwright UI if you can find a selector? we should definitely be able to select elements by their labels

the issue is really not that important, but we should take the opportunity to make sure we know how to use the library output

Reply:
Reverted in https://github.com/cpvalente/ontime/pull/1438/commits/6d010e52df0dec7ff822be33c4a516021f53d30e